lib/plausible_web/components/generic.ex

```diff
@@ -366,7 +366,7 @@ defmodule PlausibleWeb.Components.Generic do
 
   attr(:href, :string, required: true)
   attr(:new_tab, :boolean, default: false)
-  attr(:class, :string, default: nil)
+  attr(:class, :any, default: nil)
```

Comment:
An alternative approach here could be:

```suggestion
  attr(:class, :string, default: "")
```

IMHO it's slightly better as it preserves the type validation.

Reply:
Ugh, sorry, I have missed the fact that the class attribute is not "flattened" into a single string and passed down to a component 🙈  Using any here and at the other spot is the right choice then. Again, sorry for trouble.